src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py

```diff
@@ -16,6 +16,7 @@
 
 
 def register_providers():
+    _register_one_provider("Microsoft.BotService/botServices", "2021-03-01", True)
     _register_one_provider("Microsoft.Automation/automationAccounts", "2020-01-13-preview", True)
     _register_one_provider('Microsoft.AppConfiguration/configurationStores', '2020-06-01', True)
     _register_one_provider("Microsoft.Batch/batchAccounts", "2020-03-01", True)
```

Comment:
Please put it in alphabetical order.


```suggestion
    _register_one_provider("Microsoft.Automation/automationAccounts", "2020-01-13-preview", True)
    _register_one_provider('Microsoft.AppConfiguration/configurationStores', '2020-06-01', True)
    _register_one_provider("Microsoft.Batch/batchAccounts", "2020-03-01", True)
    _register_one_provider("Microsoft.BotService/botServices", "2021-03-01", True)

```